server/controllers/cohortController.js

```diff
@@ -1,6 +1,15 @@
-export const getAllCohorts = (req, res) => {
-    res.status(200).json("hello from Cohorts route");
-};
+import db from "../db";
+export const getAllCohorts = async (req, res) => {
+    try {
+      const q = "SELECT * FROM cohorts";
+      const result = await db.query(q);
+      const cohorts = result.rows; // every rows has three columns currently.
```

Comment:
No need to have this comment as it will soon get out of date

Reply:
good point